app/google-services.json

```diff
@@ -0,0 +1,39 @@
+{
```

Comment:
This file should not be in public repo. It contains private info. Add it to gitignore and remove from commit

Reply:
Fixed